pg/benchmarks/pg_scope_all.rb

```diff
@@ -0,0 +1,23 @@
+require 'bundler/setup'
+require 'pg'
+require_relative 'support/benchmark_pg'
+
+# Connect to POSTGRES database with provided values or use default
+conn = PG.connect(host: ENV.fetch("HOST", "localhost"),
+                  port: ENV.fetch("PORT", "5432"),
+                  dbname: ENV.fetch("DB_NAME", "rubybench"),
+                  user: ENV.fetch("DB_USER", "postgres"),
+                  password: ENV.fetch("DB_PASSWORD", "postgres"))
+
+conn.exec("DROP TABLE IF EXISTS users;")
+conn.exec("CREATE TABLE users ( name varchar(255), email varchar(255) );")
+
+1000.times do
+  conn.exec("INSERT INTO users VALUES ('Lorem ipsum dolor sit amet, consectetur adipiscing elit.', 'foobar@email.com');")
+end
+
+Benchmark.pg("raw_pg/postgres_scope_all", time: 5) do
+  str = ""
+  conn.exec("SELECT * FROM users").each_row{ |row| str << "name: #{row[0]} email: #{row[1]}\n" }
```

Comment:
interestingly this can be done a fraction quicker if you avoid the #each_row and instead just use a double while loop iteration cause you can avoid a bunch of block transitions, its very very minor